server/app.js

```diff
@@ -25,6 +25,7 @@ var cookieParser = require("cookie-parser");
 var flash = require("connect-flash");
 
 
+var session = require("cookie-session");
 
```

Comment:
Is session a built in tool of Express? Curious as to why it exists still, when so much else of Express has been abstracted out as separate libraries.